controllers/login.js

```diff
@@ -0,0 +1,26 @@
+const ModelUser = require('../models/user')
+const Guid = require('guid')
+
+module.exports.login = async (req, res) => {
+  try {
+    const email = req.body.email
+    const password = req.body.password
+    const User = await ModelUser.findOne({email: email})
+    if (User.email === email && User.password === password) {
+      const token = await Guid.create()
+      User.token = token
+      await User.save()
+      res.json({Authorization: token})
+    } else {
+      throw new Error()
```

Comment:
I think that you are throwing your own errors (exceptions), I'm going to prefer something like: `throw new Error("Please enter a valid credentials")`

Reply:
Great idea, thanks!